packages/lesswrong/themes/eaTheme.ts

```diff
@@ -158,6 +158,17 @@ const theme = createLWTheme({
         padding: ".7rem",
       }
     },
+    MuiMenuItem: {
+      root: {
+        '&.MuiMenuItem-selected': {
+          backgroundColor: "#c69c6a",
```

Comment:
Now we have this orange in our theme :) Let me know if you have a different color suggestion

Reply:
😍

Reply:
>(to me) orange feels "negative"

Interesting—I didn't have this association initially, but it makes sense to me. I wonder if a brighter color would help, e.g. #edc499? 
<img width="213" alt="image" src="https://user-images.githubusercontent.com/25752873/154727254-743663ee-cd62-458f-b51e-82bb4f5014d3.png">

In general I feel excited about the EA Forum having more than one color in our palette, and I think any first instance of broadening the palette is going to feel less cohesive than the alternative, but I'm pretty agnostic about this particular case.

Reply:
@Oldmantax That's fair! I think it looks good either way so I went ahead and changed the colors:

<img width="508" alt="Screen Shot 2022-02-18 at 1 23 39 PM" src="https://user-images.githubusercontent.com/9057804/154741244-f6363d2c-2060-499c-87c0-2972bc4e7fd8.png">